appcircle_ios_build_version_increment/2.0.3/component.yaml

```diff
@@ -0,0 +1,99 @@
+platform: iOS
+buildPlatform:
+displayName: Increment Build and Version Number
+description: 'Increment Build and Version Number for iOS'
+docsUrl: https://docs.appcircle.io/versioning/ios-version
+requiredComponents: "appcircle_git_clone"
+precedingComponents: 
+followingComponents: "appcircle_export_build_artifacts"
```

Comment:
> @boztopuz Since this review is very repetitive, I will not give the same feedback in the rest, but can you provide the checks for all of them?

I mentioned the above situation in the previous review, but since the `export_build_artifacts` for which I did not add a comment were not corrected, I am adding them again for each one.
=> There is no any dependencies with `appcircle_export_build_artifacts`